### ISSUE TITLE
login/Reset input fields fail do not have sufficient contrast with background

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#44]
URL: login/reset password page
User Agent: Chrome
Language: both

Issue Description: login/Reset input fields fail do not have sufficient contrast with background
Steps to reproduce:
From the CSS for the input, the colour for the border is # CCCCCC and take the background colour #FFFFFF
put these in the [Colour contrast analyzer](https://webaim.org/resources/contrastchecker/) and determine the contrast ratio 
Tool says this is 1.6:1 ratio insufficient for UI components. 
![image](https://user-images.githubusercontent.com/65728700/128419344-ad699770-10b4-46ac-8ded-2854759c797a.png)

![image](https://user-images.githubusercontent.com/65728700/128419297-16a5c507-dadc-4747-8055-af88869e0095.png)

Expected results:
Input border colour must be at least 3:1 contrast ratio.

**Comment by mmiron-HC** [comment]
removing as this is similar to WET.
